src/components/common/YourCompany/YourCompany.vue

```diff
@@ -152,9 +152,7 @@
       <!-- Name Request Applicant -->
       <v-row no-gutters v-if="(isAlterationFiling || isChangeFiling) && hasNewNr" class="sub-section">
         <v-col cols="3">
-          <v-layout column>
-            <label><strong>Name Request Applicant</strong></label>
-          </v-layout>
+          <label class="pr-4"><strong>Name Request Applicant</strong></label>
```

Comment:
Check whether we have base CSS to make all labels bold. If not, I know a lot of components have local CSS to do so.

I think we should move towards global CSS for this if we don't it already. (I can't think of when labels wouldn't be bold.)

Reply:
Will look! 👁️

Reply:
Labels were already `bolded`. 
This was redundant! Removed